src/test/java/teammates/test/cases/webapi/UpdateInstructorPrivilegeActionTest.java

```diff
@@ -0,0 +1,309 @@
+package teammates.test.cases.webapi;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.exception.InvalidHttpRequestBodyException;
+import teammates.common.util.Const;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.action.UpdateInstructorPrivilegeAction;
+import teammates.ui.webapi.output.InstructorPrivilegeData;
+import teammates.ui.webapi.output.MessageOutput;
+import teammates.ui.webapi.request.InstructorPrivilegeUpdateRequest;
+
+/**
+ * SUT: {@link UpdateInstructorPrivilegeAction}.
+ */
+public class UpdateInstructorPrivilegeActionTest extends BaseActionTest<UpdateInstructorPrivilegeAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.INSTRUCTOR_PRIVILEGE;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return PUT;
+    }
+
+    @Override
+    protected void testExecute() throws Exception {
+        // see individual tests
+    }
+
+    @Test
+    protected void testExecute_withCourseId_shouldUpdatePrivilegesInCourseLevel() {
+        InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+
+        String[] submissionParams = new String[] {
+                Const.ParamsNames.INSTRUCTOR_ID, instructor1OfCourse1.getGoogleId(),
+                Const.ParamsNames.COURSE_ID, instructor1OfCourse1.courseId,
+        };
+
+        InstructorPrivilegeUpdateRequest reqBody = new InstructorPrivilegeUpdateRequest();
+        reqBody.setCanModifyCourse(false);
+        reqBody.setCanModifySessionCommentsInSections(false);
```

Comment:
Verify that these values were changed (i.e. that their values are true before running the action). Also should probably verify that the other values aren't changed.

Reply:
emm but the values are specified in typicalBundle?

Reply:
True. I feel like this makes it clear that change is actually applied, since it's possible that the values were originally false. Also it ensures that the test fails if the typical bundle values are modified, so people know that they need to update the test here. But I'm ok with leaving it to the senor devs and seeing if they're ok with it.

Reply:
Yup. It would be a good practice to do that. Just add one/two line assertion to the old privilege.

BTW, I suppose you should change all privileges but not just two to ensure every line is covered? If we are doing this, you actually can revert the original privilege and verify the privilege is indeed reverted.